parachain/pallets/basic-channel/src/inbound/mod.rs

```diff
@@ -13,6 +13,8 @@ use artemis_core::{
 
 use envelope::Envelope;
 
+mod benchmarking;
```

Comment:
I think we should only declare the benchmarking module if the `runtime-benchmarks` is enabled.

```
#[cfg(feature = "runtime-benchmarks")]
pub mod benchmarking;
```

Reply:
> @Rizziepit hey so I overlooked that we can set the initial locked DOT balance using the genesis configuration for the balances pallet.
> 
> Given that the account id for the DOTApp module is `5EYCAe5jHEQsVPTRQqy6NCeG71Hz1EVXikZxTkr67fM8j2Rd`, we can append the following item to the `genesis.runtime.palletBalances.balances` array to initialize the locked balance to a large number.
> 
> ```json
> [
>   "5EYCAe5jHEQsVPTRQqy6NCeG71Hz1EVXikZxTkr67fM8j2Rd",
>   1152921504606846976
> ]
> ```

Nice! I was struggling to figure out what the module account ID is. How did you convert `ModuleId(*b"s/dotapp")` to account ID?

Reply:
I have used this in the past: https://github.com/paritytech/substrate/blob/9b08105b8c7106d723c4f470304ad9e2868569d9/utils/frame/frame-utilities-cli/src/module_id.rs

Reply:
Nice, didn't know about that tool. I managed to get the account id by doing the following in a unit test:

```rust
use sp_runtime::traits::AccountIdConversion;

fn print_account() {
    println!("{}", ModuleId(*b"s/dotapp").into_account())
}
```

Reply:
That‘s also super straight forward, thanks for sharing!